the-telephone-app-fe/src/components/DescribePicture/DescribePicture.js

```diff
@@ -7,7 +7,7 @@ import axios from "axios";
 import { BsFillCheckCircleFill } from "react-icons/bs";
 
 const DescribePicture = () => {
-  let ip = "http://192.168.101.180:9090/";
+  let ip = "https://garticphone-ces-intern.onrender.com/";
```

Comment:
The link should be defined as a const in a config file, then import it here and other files to use.

Reply:
Fix this later.